doc/help/CP.md

```diff
@@ -0,0 +1,7 @@
+kdb-cp(1) -- Copy keys within the key database
+================================
+
+## USAGE
+`kdb cp <source> <dest>`  
```

Comment:
Please include such information! If it was not obvious for you, it won't be 
obvious for others.